sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample5_RecognizePiiEntities.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.AI.TextAnalytics.Tests;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.TextAnalytics.Samples
+{
+    [LiveOnly]
+    public partial class TextAnalyticsSamples : SamplesBase<TextAnalyticsTestEnvironment>
+    {
+        [Test]
+        public void RecognizePiiEntities()
+        {
+            string endpoint = TestEnvironment.Endpoint;
+            string apiKey = TestEnvironment.ApiKey;
+
+            #region Snippet:TextAnalyticsSample5CreateClient
+            var client = new TextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+            #endregion
+
+            #region Snippet:RecognizePiiEntities
+            string document = "A developer with SSN 859-98-0987 whose phone number is 800-102-1100 is building tools with our APIs.";
+
+            PiiEntityCollection entities = client.RecognizePiiEntities(document).Value;
+
+            Console.WriteLine($"Redacted Text: {entities.RedactedText}");
+            Console.WriteLine($"Recognized {entities.Count} PII entit{(entities.Count > 1 ? "ies" : "y")}:");
```

Comment:
I'm just being annoying at this point, but what if there are 0 entities (is that even possible? I don't know much)? If that's a concern:
```cs
entities.Count == 1 ? "y" : "ies"
```

Reply:
Fixed.
I have this in other samples :| so I created issue https://github.com/Azure/azure-sdk-for-net/issues/14833 to fix it